lib/spack/spack/test/test_input/CMakeCache.txt

```diff
@@ -0,0 +1,447 @@
+# This is the CMakeCache file.
```

Comment:
Well, what worries me is that there are a lot of weird things in CMake cache files, I think having at least one full one would help if we decide to go down this road. On the RAJA-centricity, I really don't care what we use, I just want it to be something real